tbx/project_styleguide/templates/patterns/_pattern_library_only/streamfield/division_story_container.html

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
```suggestion
```

Reply:
Done in [60249d2](https://github.com/torchbox/torchbox.com/pull/324/commits/60249d24b02626e7005f1e96d2218144a1c714fd)